src/main/scala/mesosphere/marathon/MarathonModule.scala

```diff
@@ -75,6 +77,15 @@ class MarathonModule(conf: MarathonConf, http: HttpConf, zk: ZooKeeperClient)
 
   }
 
+  @Named(ModuleNames.NAMED_HTTP_EVENT_STREAM)
+  @Provides
+  @Singleton
+  def provideHttpEventStreamActor(system: ActorSystem,
+                                  @Named(EventModule.busName) eventBus: EventStream): ActorRef = {
+    val outstanding = 50 //yet another command line parameter??
```

Comment:
I think this is hard to decide. That we need to a configuration parameter for every minor config options sucks. But I actually could imagine that someone needs to adjust this, unfortunately.